src/fontra/client/css/tooltip.css

```diff
@@ -38,7 +38,7 @@
   color: var(--tooltip-foreground-color);
   background: var(--tooltip-background-color);
   z-index: 1;
-  opacity: 0;
+  visibility: hidden;
```

Comment:
Why this change? I think visibility is not animatable fluidly as opacity is. I'm pretty sure we have a transition.

Reply:
This is necessary, because otherwise the hidden tooltip causes issues with the toolbar. You cannot click on a toolbar item, when there is a hidden tooltip. does that make sense to you? I could demo it, if you want. But this change is definitely necessary.

Reply:
I think we have a different problem with tooltips that I just noticed.

Example: go to the fullscreen button, hover until the tooltip comes. Then click on the tooltip: it's as if you're clicking on the button itself.

Worse: hover over the fullscreen button until the tooltip appears, then hover over the tooltip until it disappears, then click: it is as if the button was clicked...

The tooltip should probably have `pointer-events: none` set.

I think your `visibility` changes tries to address this problem, but in the wrong way, as we should be able to use opacity.

Reply:
I see. I will give it a try and let you know.

Reply:
I opened a separate issue: https://github.com/googlefonts/fontra/issues/1227

Reply:
Thanks a lot for the hint with 'pointer-events: none' this fixes the tooltip issue. Done.